### PR TITLE
[High level] Whatsapp subscribe form

### PR DESCRIPTION
**Pull request by swetha-charles**
Add whatsapp page to subscribe and unsubscribe.

Changes I want to make when I'm back:
- change how the user slice works to only update whatsapp subscriptions on whatsapp endpoints. Right now, the entire activeSubscriptions field changes
- pull out subscribe and unsubscribe forms as separate components

### CODE REVIEW

File: components/forms/WhatsappForm.tsx [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+    const validatedNumber = validateNumber(phonenumber);
+
+    if (validatedNumber === undefined) {
+      setFormError(t('subscribeErrors.invalidNumber'));
+      setLoading(false);
+      return;
+    } else {
+      setPhonenumber(validatedNumber);
+
+      const subscribeResponse = await subscribeToWhatsapp({
```
Comment (eleanorreem) [inline components/forms/WhatsappForm.tsx:72]: Not 100% but do these need to be wrapped in try catches?

File: components/forms/WhatsappForm.tsx [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    }
+  };
+
+  const unsubscribeHandler = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setFormError('');
+    setLoading(true);
+    logEvent(WHATSAPP_UNSUBSCRIBE_REQUEST);
+
+    const unsubscribeResponse = await unsubscribeFromWhatsapp({
```
Comment (eleanorreem) [inline components/forms/WhatsappForm.tsx:113]: again here - do these not need try catch?

File: messages/whatsapp/pt.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (eleanorreem) [inline messages/whatsapp/pt.json:1]: On this note - make sure you prioritise a translation ticket!

Comment (swetha-charles) [reply messages/whatsapp/pt.json:1]: Good point - have added ticket on notion board for this

File: components/forms/WhatsappForm.tsx (hunk 1 above)
Comment (swetha-charles) [reply components/forms/WhatsappForm.tsx:72]: Have written below.

**Pull request merged**